python/sbp/client/util/settingmonitor.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (C) 2017 Swift Navigation Inc.
+# Contact: Swift Navigation <dev@swiftnav.com>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+"""
+the :mod:`sbp.client.util.settingmonitor` module contains functionality to
+monitor SBP_MSG_SETTINGS_READ_RESP messages for desired values
+"""
+
+import time
+
+class SettingMonitor(object):
+    def __init__(self):
+        self.settings = []
+
+    def capture_setting(self, sbp_msg, **metadata):
```

Comment:
While I think this level of integration is fine for this utility, I think it would be a bit confusing for someone to use without a comment explaining the usage or an example. Alternatively if this Monitor handled the registering the callback I think it might be more self explanatory.